index.html

```diff
@@ -63,6 +67,184 @@ <h1 class="item">Hey There. I’m Mumenya</h1>
           </ul>
         </div>
       </section>
+      <section>
+        <div id="item1">
```

Comment:
- [ ] For better practices you can use semantic stages here e.g. ```<article>```, please try to minimalize the use of ```<div>``` in your code instead use ```<section>``` and ```<article>```